frontend/src/features/contacts/contact/Person/OrganizationSubForm.tsx

```diff
@@ -0,0 +1,61 @@
+import { Input } from 'components/common/form';
```

Comment:
why is this file (orgsubform) under the **Person** folder? Wouldn't it make sense to put it under Organization?

Reply:
Also, the Person sub-form includes the 3 addresses but this one doesn't - any reason for that?